python/CMakeLists.txt

```diff
@@ -1 +1,17 @@
+# Python cmake, if testing enabled set pip to install test dependencies in venv
+if (KWIVER_ENABLE_TESTS)
+    if (PIP_COMMAND)
+        list(SUBLIST PIP_COMMAND 0 1 py_pip_command)
+        list(SUBLIST PIP_COMMAND 1 -1 pip_args)
+        python_target_add_command( "pip-install-upgrade"
+                                   ${py_pip_command}
```

Comment:
So `py_pip_command` is just `python3_EXECUTABLE`, and `pip_args` is the upgrading args? It might be worth restructuring the variables in `CMake/kwiver-setup-python.cmake` to something like:

`set(pip_command "${python3_EXECUTABLE}" "-m" "pip")`

`set(pip_upgrade_command "${pip_command}" "-q" "install" "--upgrade" "pip")`

`set(pip_dep_install "${pip_command}" "-q"
                        "install"
                        "-r"
                        "${KWIVER_SOURCE_DIR}/python/requirements.txt"
                        "||"
                        "${CMAKE_COMMAND}" "-E" "echo"
                        "Pip install failed, consult build output. Python dependencies may not be met"`


Then in `python/CMakeLists.txt`:

```
# Python cmake, if testing enabled set pip to install test dependencies in venv
if (KWIVER_ENABLE_TESTS)
    if (PIP_COMMAND)
        python_target_add_command( "pip-install-upgrade"
                                   ${pip_upgrade_command}
                                   "Upgrading pip...")
        python_target_add_command( "pip-install-deps"
                                   ${pip_dep_install}
                                   "${pip_args}"
                                   "Pip installing Python dependencies..."
                                   "pip-install-upgrade")
    endif()
endif()
```